core/Cargo.toml

```diff
@@ -7,16 +7,18 @@ description = "The code generator used by Typeshare's command line tool"
 repository = "https://github.com/1Password/typeshare"
 
 [dependencies]
+anyhow.workspace = true
```

Comment:
For my own curiosity, was there something in these changes that necessitated moving `anyhow` to be a regular dependency, rather than a dev dependency?

Reply:
Bot the `cli` and `core` crates are using it so any common dependency I'll move to the workspace decencies.

Reply:
The workspace part makes sense, I was specifically asking about it moving from being under `dev-dependencies` to being under `dependencies`. Does `workspace` not work in `dev-dependencies`?

Reply:
Updated